qa/integration/framework/fixture.rb

```diff
@@ -66,16 +66,7 @@ def get_service(name)
     @service_locator.get_service(name)
   end
 
-  def output_equals_expected?
-    FileUtils.identical?(@actual_output, @input)
-  end
-
-  def output_exists?
-    File.exists?(@actual_output)
-  end
-
   def teardown
-    File.delete(@actual_output) if @settings.is_set?("actual_output") && output_exists?
```

Comment:
the cleanup was moved to the spec -> as it had assumptions about current work-dir.
(the relative path is considered LS_HOME relative - this is now explicit in the one spec using `actual_output: ...`)